packages/sdk/src/types/messages/ConnectorMessage.ts

```diff
@@ -1,9 +1,9 @@
-import { ConnectorMessageContentDerivation } from "./ConnectorMessageContentDerivation";
+import { ArbitraryMessageContentJSON, MailJSON, NotificationJSON, RequestJSON, ResponseWrapperJSON } from "@nmshd/content";
 import { ConnectorMessageRecipient } from "./ConnectorMessageRecipient";
 
 export interface ConnectorMessage {
     id: string;
-    content: ConnectorMessageContentDerivation;
+    content: MailJSON | ResponseWrapperJSON | RequestJSON | NotificationJSON | ArbitraryMessageContentJSON;
```

Comment:
We could still define `MessageContentDerivation = MailJSON | ResponseWrapperJSON | RequestJSON | NotificationJSON | ArbitraryMessageContentJSON`, since it is used in multiple places.

Reply:
I thought about that, but would expect that to be done in content, so we can do that if content provides it.

Reply:
True, I guess this should already happen in content and not only defining the DTOs. Should we quickly slide in a Runtime PR? :D

Reply:
I think this will automatically drop in when we have the DTOs. Content should not define the derivations - the runtime should.